docs/content/reference/command-line-interface/_index.md

```diff
@@ -208,9 +209,9 @@ drasi completion zsh -h
 **Purpose**: The `delete` command is used to delete one or more Drasi resources.
 
 **Arguments and Flags**:
-- `<resource-type> <resource-id>`: specifies the type and the ID of the resource to delete.
+- `<resource-type> <resource-id>` (required): specifies the type and the ID of the resource to delete.
```

Comment:
This is not required if you give it yaml files or pipe yaml into it.

Reply:
Removed required/optional info for flags/options